### DISCUSSION TITLE
Receiving OAuthCallbackError when trying to sign in with different account.

### DISCUSSION THREAD

**Discussion by RamsayRomero** [nextauthjs/next-auth#7614] [Help]
I'm using NextAuth with the Spotify provider. Everything has been working as expected when testing with my account. I created a second Spotify account and tried to log in with that account, but it doesn't work. I am redirected to the default signin page with an error message that says "Try signing in with a different account." My console shows a 'OAuthCallbackError' with a message: 'expected 200 OK, got: 403 Forbidden'. I tried clearing my cookies and local storage and resetting my database, but I am still getting this error when trying to sign in with my second Spotify account. Signing in with my original Spotify account works normally. Does anyone have an idea of what could be causing this issue?

**Comment by OatsProgramming** [comment]
Same here. Think its a legitimate issue with NextAuth. There's quite a few ppl that have said the same thing for other providers

**Comment by jdbadilla** [comment]
Have you been able to find an answer? I am struggling with the exact same issue you have described @RamsayRomero .

**Reply by jdbadilla** [reply]
or you @OatsProgramming

**Comment by jdbadilla** [comment]
Ok I see what the problem is: you have to add more users in the Spotify development portal so that they may access it...
